pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -214,6 +214,10 @@ stdenv.mkDerivation ({
 
   LANG = "en_US.UTF-8";         # GHC needs the locale configured during the Haddock phase.
 
+  # XXX: workaround for https://ghc.haskell.org/trac/ghc/ticket/11042.
+  LD_LIBRARY_PATH = stdenv.lib.makeLibraryPath (stdenv.lib.filter (x: x != null) systemBuildInputs);
+                    # ^^^ Internally uses `getOutput "lib"` (equiv. to getLib)
```

Comment:
Is this comment is pointing out the definition of the function or is it something special?